robot_mover_node:
  ros__parameters:
    node_rate: 4.0
    velocity_topic: "commands/velocity"
    linear_vel: 0.3
    angular_vel: 0.5
    turning_time: 2.0                       # Seconds
    max_loss_time: 1.0                      # Seconds. [1.0 - inf]

collision_detector_node:
  ros__parameters:
    node_rate: 5.0
    laser_topic: "scan"
    left_range: [0.392, 0.785]
    right_range: [5.497, 5.890]
    front_range: [5.890, 6.675]
    fixed_frame: "base_scan"
    child_frame: "collision_frame"
    frame_transform: [0.0, 0.0, 0.0, 0.0, 0.0, 0.0]